src/utils/scripts.ts

```diff
@@ -14,8 +18,67 @@ export async function runScript(
     }
 
     for (const script of scripts) {
-        // TODO: use execa instead of runNewProcess
         const success = await runNewProcess(script, cwd);
         if (!success) throw new UserError(`Failed to run script: ${script}`);
     }
 }
+
+export async function runFrontendStartScript(
+    scripts: string | string[] | undefined,
+    cwd: string,
+): Promise<void> {
+    if (!scripts) {
+        return;
+    }
+
+    if (!Array.isArray(scripts)) {
+        scripts = [scripts];
+    }
+
+    for (const script of scripts) {
+        await new Promise<void>((resolve, reject) => {
+            const child = spawn(script, { cwd, shell: true, stdio: "pipe" });
+
+            let debounceCount = 0;
+            let logsBuffer: Buffer[] = [];
+            // A debounce is needed when logs are printed on `data` event because the logs are printed in chunks.
+            // We want to print chunkc of logs together if they are printed within a short time frame.
+            const debouncedLog = _.debounce(() => {
+                debounceCount = 0;
+                if (logsBuffer.length === 0) return;
+
+                const logs = Buffer.concat(logsBuffer).toString().trim();
+                log.info(
+                    `${colors.magenta("[Frontend logs]\n| ")}${logs.split("\n").join(`\n${colors.magenta("| ")}`)}`,
+                );
+                logsBuffer = [];
```

Comment:
In case of multiple frontends, shouldn't we display an index for each one and assign a different color?

Reply:
https://github.com/Genez-io/genezio/pull/949